b2sdk/v1/file_version.py

```diff
@@ -8,16 +8,76 @@
 #
 ######################################################################
 
+from typing import Optional
 import datetime
 import functools
 
 from b2sdk import _v2 as v2
+from ..raw_api import SRC_LAST_MODIFIED_MILLIS
+from . import api as v1api
 
 
-# override to retain old formatting methods
+# Override to retain legacy class name, __init__ signature, slots
+# and old formatting methods
 class FileVersionInfo(v2.FileVersion):
+    __slots__ = [
```

Comment:
> The action of a __slots__ declaration is not limited to the class where it is defined. __slots__ declared in parents are available in child classes. However, child subclasses will get a __dict__ and __weakref__ unless they also define __slots__ (which should only contain names of any additional slots).

> If a class defines a slot also defined in a base class, the instance variable defined by the base class slot is inaccessible (except by retrieving its descriptor directly from the base class). This renders the meaning of the program undefined. In the future, a check may be added to prevent this.

https://docs.python.org/3/reference/datamodel.html#notes-on-using-slots